keyvi/src/cpp/dictionary/dictionary.h

```diff
@@ -70,7 +70,7 @@ final {
     }
 
     // temporary implementation
-    fsa::automata_t GetFsa() {
+    fsa::automata_t GetFsa() const {
```

Comment:
hmm, there is no internal state in Automata. So the only state would be the shared pointer itself. I will try it.